clientupdate/clientupdate.go

```diff
@@ -119,11 +126,14 @@ type Updater struct {
 	// Update is a platform-specific method that updates the installation. May be
 	// nil (not all platforms support updates from within Tailscale).
 	Update func() error
+
+	currentVersion string
```

Comment:
(Optional) Can we export this too? The value is built by the constructor anyway, and it'd be nice to have it doc'd (in particular what shape the version is meant to have)

Reply:
I added this as a field for testing purposes only. External callers shouldn't need to set this explicitly. 
Added the docs :+1: